public/pages/CreateMonitor/containers/MonitorRoles/MonitorRoles.js

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+import { EuiHealth, EuiHighlight } from '@elastic/eui';
+
+import { FormikComboBox } from '../../../../components/FormControls';
+import { createReasonableWait } from '../MonitorIndex/utils/helpers';
+
+const propTypes = {
+  httpClient: PropTypes.object.isRequired,
+};
+
+class MonitorRoles extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.lastQuery = null;
+    this.state = {
+      isLoading: false,
+      showingRoleQueryErrors: false,
+      options: [],
+      roles: [],
+    };
+
+    this.onSearchChange = this.onSearchChange.bind(this);
+    this.handleQueryRoles = this.handleQueryRoles.bind(this);
+    this.onFetch = this.onFetch.bind(this);
+  }
+
+  componentDidMount() {
+    // Simulate initial load.
+    this.onSearchChange('');
+  }
+
+  async onSearchChange(searchValue) {
+    let query = searchValue;
+    this.lastQuery = query;
+    this.setState({ query, showingRoleQueryErrors: !!query.length });
+
+    await this.onFetch(query);
+  }
+
+  async handleQueryRoles(rawRole) {
+    const role = rawRole.trim();
+
+    try {
+      const response = await this.props.httpClient.post('../api/alerting/_roles', {
+        body: JSON.stringify({ role }),
+      });
+      if (response.ok) {
+        const roles = response.resp.map((role) => ({
+          label: role,
+          role,
+        }));
+        return _.sortBy(roles, 'label');
+      }
+      return [];
+    } catch (err) {
+      console.error(err);
+      return [];
+    }
+  }
+
+  async onFetch(query) {
+    this.setState({ isLoading: true });
+    const roles = await this.handleQueryRoles(query);
+    createReasonableWait(() => {
+      // If the search changed, discard this state
+      if (query !== this.lastQuery) {
+        return;
+      }
+      this.setState({ roles, isLoading: false });
+    });
+  }
+
+  render() {
+    const { roles, isLoading } = this.state;
+
+    const visibleOptions = [
+      {
+        label: 'Backend roles',
+        options: roles,
+      },
+    ];
+
+    return (
+      <FormikComboBox
+        name="roles"
+        formRow
+        rowProps={{
+          label: 'Backend roles',
```

Comment:
Discussed the UI element with our UI developer. Could you change the `label` to the following, and remove the `helpText`?
```
label: (
            <div>
              <EuiText size={'xs'}>
                <strong>Backend roles</strong>
                <i> - optional </i>
              </EuiText>
              <EuiText color={'subdued'} size={'xs'} style={{ width: '400px' }}>
                Specify role-based access control (RBAC) backend roles.{' '}
                <EuiLink external href={'https://opensearch.org/docs/latest/observing-your-data/alerting/security/'} target={'_blank'}>
                  Learn more
                </EuiLink>
              </EuiText>
            </div>
          ),